Utilities/wearable-packager/marketplace-packager.js

```diff
@@ -0,0 +1,144 @@
+//
+//  marketplace-packager.js
+//
+//  A utility script for packaging up an item to generate a properly-configured JSON
+//  for releasing to Marketplace as a wearable item.
+// 
+//  Created by Liv Erickson on 11/16/17
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+/* globals Selection */
+
+(function(){
+    var APP_NAME = "WEARPKGR";
+    var APP_URL = Script.resolvePath("app.html");
+    var APP_ICON = Script.resolvePath("icon.png");
+
+    var prevID = 0;
+    var listName = "contextOverlayHighlightList";
+    var listType = "entity";
+
+    var entityIDToExport = "";
+
+    var tablet = Tablet.getTablet('com.highfidelity.interface.tablet.system');    
+
+    function handleMousePress(entityID) {
+        if (prevID !== entityID) {
+            Selection.addToSelectedItemsList(listName, listType, entityID);
+            prevID = entityID;
+        }
+        tablet.emitScriptEvent(entityID);
+    }
+
+    function handleMouseLeave(entityID) {
+        if (prevID !== 0) {
+            Selection.removeFromSelectedItemsList("contextOverlayHighlightList", listType, prevID);
+            prevID = 0;
+        }
+    }
+
+    var baseUserdata = {
+        "Attachment": {
+            "action": "attach",
+            "joint": "Hips",
+            "attached" : false,
+            "options": {
+                "translation": {
+                    "x": 0,
+                    "y": 0,
+                    "z": 0
+                },
+                "scale": 1
+            }
+        },
+        "grabbableKey": {
+            "cloneable": false,
+            "grabbable": true
+        }
+    };
+
+    var exportProperties = {
+        type: "Model",
+        clientOnly: 1,
+        parentID: "{00000000-0000-0000-0000-000000000001}",
```

Comment:
"{00000000-0000-0000-0000-000000000001}" -> MyAvatar.sessionUUID    // that should still work, "{00000000-0000-0000-0000-000000000001}" is a client-side hack which should not be exposed to much in scripts to prevent bad practice.